test/extended/controller_manager/pull_secret.go

```diff
@@ -129,7 +129,7 @@ var _ = g.Describe("[sig-devex][Feature:OpenShiftControllerManager]", func() {
 		}
 	})
 
-	g.It("TestDockercfgTokenDeletedController", func() {
+	g.It("TestDockercfgTokenDeletedController [apigroup:image.openshift.io]", func() {
```

Comment:
Where does the image apigroup requirement come from? `waitForServiceAccountPullSecret`?

Reply:
Yes, `waitForServiceAccountPullSecret` expects a secret to be provisioned in order to make use of the image API.

Reply:
Is it possible to say by which component is the secret provisioned?

Reply:
That would be the openshift controller manager.

Reply:
TestAutomaticCreationOfPullSecrets follows the same workflow with the default sa name. Checking for presence of image/cluster object.

Reply:
That test is updated too: https://github.com/openshift/origin/pull/27392/files#diff-e548b9ac6cbe0bee7cc02da7c4b6ced6eb44801339dfea05220e4285b7eec558R63
Or did I misunderstand you?

Reply:
I was just justifying why adding image api group into `TestDockercfgTokenDeletedController` is required. The test itself does not give any indication why the image group is needed. Though, following `TestAutomaticCreationOfPullSecrets` gives some idea.

Reply:
Oh ok, didnt follow you. Thanks!